en/components/navdrawer.md

```diff
@@ -316,6 +316,65 @@ After all the steps above are completed, your app should look like that:
 </code-view>
 
 
+<div class="divider--half"></div>
+
+## Hierarchical Navigation
+
+To create a multi-level hierarchical navigation using the `IgxNavigationDrawerComponent`, you can use the `[IgxTreeComponent](tree.md)` in the `igxDrawer` template. The tree can be constructed directly from your application `Routes` object. Here's how this can be acieved:
+
+```html
+<igx-nav-drawer [isOpen]="true" [enableGestures]="true" width="280px">
+    <ng-template igxDrawer>
+        <igx-tree>
+            <igx-tree-node *ngFor="let route of routes">
+                <a igxTreeNodeLink [routerLink]="route.path">{{ route.data?.displayName }}</a>
```

Comment:
@damyanpetev Setting `routerLinkActive` works on the tree.

Reply:
I wouldn't expect otherwise :) Point being, setting `routerLinkActive="route-selected-class"` as shown is shorted but also required the user to write their own selected styles - i.e. not reusing the ones we already provide and the active route node will not be reflected in the tree API as selected too. It's an option, just not sure which is best. That's why I suggested a similar approach to the existing routing section where the directive state is assigned back the the item.

Reply:
@damyanpetev I've used this approach in the top-level navigation. I can update the doc at a later stage if we get a request for an example.